### DISCUSSION TITLE
pilotnet driving dataset availability for pilotnet_snn notebook

### DISCUSSION THREAD

**Discussion by gwgkarunasena57** [lava-nc/lava-dl#314] [Q&A]
Hello,

I am trying the pilotnet based notebook, but the dataset is not available in it's original location. If anyone already has pilotnet dataset (driving_dataset.zip) please share a possible link to download.
Thank you,

Thanks and Rgds,
Udayanga

**Comment by Michaeljurado42** [comment]
Think it might be broken on windows.

Cheers.
https://docs.google.com/uc?export=download&id=1Ue4XohCOV5YXy57S_5tDfCVqzLr101M7

**Comment by gwgkarunasena57** [comment]
Thank you Michael !
